test-harness/build-base-image.sh

```diff
@@ -1,4 +1,5 @@
-#!/bin/bash
+#!/usr/bin/env bash
+#/usr⁩/local⁩/⁨Cellar⁩/gnu-getopt⁩/2.33.2⁩/bin⁩/getopt
```

Comment:
Is this line doing anything under the hood? To my untrained bash eye, it seems to be a comment only. Is there some pre-procssing magic that happens here?

If not, is it safe to remove?

Reply:
Yup, that line was a mistake that should not have made it into the PR. Will remove it.